acra/acra-in-depth/architecture/acraserver.md

```diff
@@ -5,12 +5,19 @@ weight: 2
 
 # AcraServer, an SQL database proxy
 
-AcraServer is also known as SQL Proxy. It's SQL database proxy that exposes Acra’s functionality by parsing SQL traffic between an app and a database and applying security functions where appropriate. 
+## How AcraServer works
 
-If you'd like to encrypt the data between your app and SQL database "transparently on-a-fly" then AcraServer should be your main choice.
+AcraServer is database protocol aware proxy between application and MySQL/PostgreSQL that does next:
```

Comment:
```suggestion
AcraServer is also known as SQL Proxy. It's an SQL database proxy that exposes Acra’s functionality by parsing SQL traffic between an app and a database and applying security functions where appropriate. 

If you'd like to encrypt the data between your app and SQL database "transparently on-a-fly" then AcraServer should be your main choice.

AcraServer sits between your application and MySQL/PostgreSQL database and does the following:
```

Reply:
heh, I decided to remove these big paragraph and simplify, but you decided back it)